PerlMagick/Makefile.PL.in

```diff
@@ -173,7 +173,7 @@ if (($^O eq 'MSWin32') && ($Config{cc} =~ /gcc/)) {
   #
   # Setup for strawberry perl.
   #
-  $INC_magick       = "$Ipaths";
+  $INC_magick       = "$Ipaths @CPPFLAGS@";
```

Comment:
I checked now on Windows 11, MSYS2. The `configure` script (`config.status`) substitues the following for `@CPPFLAGS`

```
-DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_CHANNEL_MASK_DEPTH=32 -DMAGICKCORE_QUANTUM_DEPTH=16 -D_DLL -D_MT
```
Do we want the last two flags `-D_DLL -D_MT` ? If not, we should find another substitution that only includes the defines: `-DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_CHANNEL_MASK_DEPTH=32 -DMAGICKCORE_QUANTUM_DEPTH=16`